internal/moreos/testdata/fake_func-e.go

```diff
@@ -56,6 +56,10 @@ func main() {
 
 	// Block until we receive SIGINT or are canceled because Envoy has died.
 	<-sigCtx.Done()
+	stop()
+
+	// Simulate handleShutdown like in envoy.Run.
+	_ = moreos.Interrupt(cmd.Process)
```

Comment:
Probably this one causing -1073741510 on Windows? If we don't have this, we waste 5 secs on waiting the child process to terminate.

Reply:
This is because I didn't set the `cmd.SysProcAttr` for windows.